src/index.ts

```diff
@@ -58,14 +58,19 @@ const defaultOptions: Partial<PinoSentryOptions> = {
   expectPinoConfig: false,
 };
 
+const defaultSentryOptions: NodeOptions = {
+  dsn: process.env.SENTRY_DSN,
```

Comment:
why take the dsn from env is needed? from the docs Sentry does that automatically
https://docs.sentry.io/concepts/key-terms/dsn-explainer/